### PR TITLE
Add the background jobs after the table was updated

### PR DESCRIPTION
**Pull request by nickvergessen**
Fixes an exception when updating
### Steps
1. `git checkout master@{2016-05-01}`
2. Install
3. `git checkout master`
4. `occ upgrade`
### Before

```
Exception: An exception occurred while executing 'INSERT INTO `oc_jobs` (`class`, `argument`, `last_run`, `last_checked`) VALUES(?, ?, ?, ?)' with params ["\\OC\\Authentication\\Token\\DefaultTokenCleanupJob", "null", 0, 1464018638]:

SQLSTATE[42S22]: Column not found: 1054 Unknown column 'last_checked' in 'field list'
Update failed
```
### After

```
Update successful
```

@PVince81 @MorrisJobke

### CODE REVIEW

File: lib/private/Updater.php [hunk 1]
```diff
@@ -243,6 +241,13 @@ private function doUpgrade($currentVersion, $installedVersion) {
 		if ($this->updateStepEnabled) {
 			$this->doCoreUpgrade();
 
+			try {
+				// TODO: replace with the new repair step mechanism https://github.com/owncloud/core/pull/24378
+				Setup::installBackgroundJobs();
+			} catch (\Exception $e) {
+				throw new \Exception($e->getMessage());
```
Comment (ChristophWurst) [inline lib/private/Updater.php]: What exactly is this magic doing? Wrapping the original exception message in a new exception?

Comment (nickvergessen) [inline lib/private/Updater.php]: dont know, copy pasting from above...

**Comment by nickvergessen** [comment]
@ppaysant

**Comment by ppaysant** [comment]
:+1: seems ok for me. try / catch probably useless

**Comment by PVince81** [comment]
Test failure is the same as on master -> merging

File: lib/private/Updater.php (hunk 1 above)
Comment (MorrisJobke) [inline lib/private/Updater.php]: I guess this only removes all the specific exceptions to the general one: `\Exception`. This more or less removes only special exception types.

**Pull request merged**